eng/Xunit3/Microsoft.Testing.Platform.targets

```diff
@@ -0,0 +1,91 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```

Comment:
this is src\Microsoft.DotNet.Arcade.Sdk\tools\Microsoft.Testing.Platform.targets from Arcade? here temporarily?

Reply:
It will be supported in Arcade, hopefully. My PR there is still WIP.

The whole `eng/Xunit3` directory should be removable once everything is in Arcade and you move to .NET 10